.mergify.yml

```diff
@@ -72,6 +72,22 @@ pull_request_rules:
     actions:
       queue:
 
+  - name: automatic merge for python version bump
+    conditions:
+      - and: *CheckRuns
+      - check-success=semantic-pull-request
+      - check-success=constraints
+      - author=mergify-ci-bot
+      - label!=work-in-progress
+      - label!=manual merge
+      - "title~=^chore: bump python version"
```

Comment:
I would replace this with `head: python-version-update`

Reply:
True, added it. I let the title check as well just for more security